tests/Auth/bot-authentication/Startup.cs

```diff
@@ -44,8 +45,10 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                 app.UseDeveloperExceptionPage();
             }
 
+            //.UseWebSockets()
```

Comment:
This can be removed

Reply:
this is a debugging test project - swap these lines in and out for debugging